src/main/java/io/inaam/main/service/security/SecurityConstants.java

```diff
@@ -0,0 +1,13 @@
+package io.inaam.main.service.security;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
```

Comment:
```suggestion
@UtilityClass
```
utility class cannot be instantiated and members are marked static implicitly.